plugins/woocommerce-blocks/assets/js/blocks/product-filters/types.ts

```diff
@@ -45,3 +45,10 @@ export type FilterBlockContext = {
 		} >;
 	};
 };
+
+export type Color = {
+	slug: string;
+	class: string;
+	name: string;
```

Comment:
These fields can be undefined when the color is a custom one.

Reply:
779b0d5627443adfd65624622f80d652f45d1466